test/lib/testMayaUsdLayerEditorCommands.py

```diff
@@ -362,6 +362,40 @@ def setAndRemoveEditTargetRecursive(layer, index, editLayer):
         cmds.undo()
         self.assertEqual(path.normcase(stage.GetEditTarget().GetLayer().identifier), sharedLayer)
 
+    def testRemoveMultipleLayers(self):
+
+        shapePath, stage = getCleanMayaStage()
+        rootLayer = stage.GetRootLayer()
+        rootLayerID = rootLayer.identifier
+
+        # Add six layers
+        layerNames = []
+        for index in range(1,7):
+            layerName = "layer%d" % index
+            layerName = cmds.mayaUsdLayerEditor(rootLayer.identifier, edit=True,  addAnonymous=layerName)
+            layerNames.append(layerName[0])
+
+        # Verify that the root has the expected six layers.
+        def subLayerNames():
+            return list(rootLayer.subLayerPaths)
+
+        def verifySubLayers(expectedNames):
+            rootSubLayers = subLayerNames()
+            self.assertEqual(len(expectedNames), len(rootSubLayers))
+            for name in expectedNames:
+                self.assertIn(name, rootSubLayers)
+
+        verifySubLayers(layerNames)
+
+        # Remove layers 0 and 1 and verify that the correct layers have been removed.
+        #
+        # Note: passing the same argument multiple times does not work in Python, so we call MEL.
+        #
+        # Note: when inserted in USD, layers are added at the top, so layer 0 and 1 are
+        #       layer5 and layer6.
+        mel.eval("""mayaUsdLayerEditor -removeSubPath 0 "%s" -removeSubPath 1 "%s" "%s";""" % (shapePath, shapePath, rootLayer.identifier))
```

Comment:
To call this directly from python try:
```suggestion
        cmds.mayaUsdLayerEditor(rootLayer.identifier, removeSubPath=((0, shapePath), (1, shapePath)))
```